scmtfvc.properties

```diff
@@ -37,3 +36,9 @@ defaults.mavenArtifactId=sonar-scm-tfvc-plugin
 2.1.2.downloadUrl=https://github.com/SonarQubeCommunity/sonar-scm-tfvc/releases/download/2.1.2/sonar-scm-tfvc-plugin-2.1.2.jar
 2.1.2.date=2015-05-02
 2.1.2.changelogUrl=https://github.com/SonarQubeCommunity/sonar-scm-tfvc/issues?q=milestone%3A2.1.2
+
+2.2.description=Fix compatibility for SQ 7.3 and above
```

Comment:
As a side note, we've started putting new versions at the top of the list rather than the bottom. Completely optional, but it makes it a little easier to do the edits when all the pieces are right there together in your text editor. :-)

Reply:
done